### DISCUSSION TITLE
Stylizations

### DISCUSSION THREAD

**Discussion by ghost** [eclipse-birt/birt#1036] [Q&A]
Hello gentlemen!

I have a question, how can I make the tables created from BIRT have a more current look?

Is there also some CSS framework that can be used to make the work easier?

I searched on the internet and didn't find satisfactory material.

I kindly ask for your help.

**Comment by speckyspooky** [comment]
Hello to you,

currently you can only use the existing CSS-stylings which are includes in the BIRT-environment.
The good thing on it is all avaulable CSS-changes will work on the outputs of HTML, PDF, etc.

I you want to change BIRT-table-style only for the HTML-viewer you can use the Text-element in HTML-mode
and there you can add via JavaScript CSS-classes of styling via JavaScript.

But be aware the changes of the HTML-viewer won't be work for the other ouput options
because only the browser will work for you to optimize the styling. The change won't be included or work on an export e.g. PDF

Kind regards
